concordia/templates/home.html

```diff
@@ -51,49 +51,48 @@ <h4 class="offwhite-text mt-half">REVIEW</h4>
+                {% else %}
+                <div class="col-12 col-md-6 mb-default text-center border-left-home-contribute">
+                    {% endif %}
+                    <a href="{% url 'transcriptions:campaign' campaign.slug %}">
+                        {% if 'Lincoln' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/AbrahamLincoln.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Rosa Parks' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/rosa-parks-campaign.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Clara Barton' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/clara-barton-campaign.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Branch Rickey' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/branch-rickey-campaign.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Mary Church Terrell' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/MaryChurchTerrell.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Bourne' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/WilliamOlandBourne.svg' %}" alt="{{ campaign.title }} image">
+                        {% else %}
+                        {% comment %} FIXME: when we implement campaign media, we also want a simple way to get the MEDIA_URL-relative thumbnail {% endcomment %}
+                        <img class="img-fluid" alt="{{ campaign.title }} image" src="{% if campaign.s3_storage %}{{ campaign.media_url }}{% else %}{{ MEDIA_URL }}{{ campaign.media_url }}{% endif %}">
+                        {% endif %}
+
+                        <h3 class="mt-half">{{ campaign.title }}</h3>
+                </div>
+                {% endfor %}
             </div>
-            <div class="col-12 col-md-4 mb-default text-center border-left-home-contribute">
-                <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/WilliamOlandBourne.svg' %}"
-                    alt="William Oland Bourne Papers">
-                <h3 class="mt-half">William Oland Bourne Papers</h3>
-                <a class="btn btn-outline-primary" href="#" role="button">Get Started!</a>
+            <div class="col mx-auto d-flex justify-content-center my-default">
+                <a class="btn btn-primary" href="{% url 'transcriptions:campaigns' %}" role="button">Browse All Campaigns</a>
             </div>
         </div>
-        <div class="col mx-auto d-flex justify-content-center my-default">
```

Comment:
I think this should be reverted – it causes the browse all button to go up to the collections row on wide screens

Reply:
I'm not able to reproduce that. How wide of a screen do you have?

Reply:
This was fixed in https://github.com/LibraryOfCongress/concordia/commit/437d83748aa45a54564699adcbbfa9bc72d42d83 (and possibly other commits) by moving the browse all button outside of the same flex container as the campaigns.